### PR TITLE
chore: update Subscription container docs

### PR DESCRIPTION
**Pull request by henrytao-me**
Ref https://github.com/Shopify/app-extension-libs/issues/1087

### CODE REVIEW

File: docs/ExtensionPoints/ProductSubscription/README.md [hunk 1]
```diff
@@ -131,7 +131,6 @@ extend(ExtensionPoint.SubscriptionManagementAdd, (root, api) => {
     onAction: () => {
       console.log('Added');
       done();
```
Comment (vividviolet) [inline docs/ExtensionPoints/ProductSubscription/README.md:133]: Same comment as the other PR, the description for `done` needs to say that the container will be closed automatically

**Review by vividviolet — CHANGES_REQUESTED** [review]

**Review by vividviolet — CHANGES_REQUESTED** [review]
@henrytao-me we need to make changes the Simulator host to call close on `done`

**Comment by henrytao-me** [comment]
Good catch.

**Comment by henrytao-me** [comment]
🎩-ed locally with https://github.com/Shopify/argo-admin-cli/pull/36. The AppOverlay and Modal are closed properly after calling `done`

**Comment by henrytao-me** [comment]
🎩-ed again after argo-admin-cli releases 0.1.11. LGTM

**Review by vividviolet — APPROVED** [review]

**Pull request merged**